src/main/java/seedu/sellsavvy/logic/parser/AddOrderCommandParser.java

```diff
@@ -0,0 +1,65 @@
+package seedu.sellsavvy.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.sellsavvy.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.sellsavvy.logic.parser.CliSyntax.PREFIX_COUNT;
+import static seedu.sellsavvy.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.sellsavvy.logic.parser.CliSyntax.PREFIX_ITEM;
+
+import java.util.stream.Stream;
+
+import seedu.sellsavvy.commons.core.index.Index;
+import seedu.sellsavvy.logic.commands.ordercommands.AddOrderCommand;
+import seedu.sellsavvy.logic.parser.exceptions.ParseException;
+import seedu.sellsavvy.model.order.Count;
+import seedu.sellsavvy.model.order.Date;
+import seedu.sellsavvy.model.order.Item;
+import seedu.sellsavvy.model.order.Order;
+
+/**
+ * Parses input arguments and creates a new AddOrderCommand object.
```

Comment:
All the other personCommand top javadoc doesn't end with a period. But I feel that it is better to change them than this.

Reply:
Some of the parser classes don't end with period so it's a bit inconsistent. Anyway, I added periods for class header comments for a few of the parser classes.